smithy-protocols-traits/src/main/resources/META-INF/services/software.amazon.smithy.model.validation.Validator

```diff
@@ -0,0 +1 @@
+software.amazon.smithy.aws.traits.protocols.ProtocolHttpValidator
```

Comment:
Should remove this file as well.

Reply:
Oh, I thought we could just piggy-back on this validator for http / httpStreaming. I'll remove it.